h2/stream.py

```diff
@@ -630,22 +699,36 @@ def send_alt_svc(self, previous_state):
         (H2StreamStateMachine.window_updated, StreamState.CLOSED),
     (StreamState.CLOSED, StreamInputs.RECV_RST_STREAM):
         (None, StreamState.CLOSED),  # Swallow further RST_STREAMs
+    (StreamState.CLOSED, StreamInputs.RECV_END_STREAM):
+        (None, StreamState.CLOSED),
     (StreamState.CLOSED, StreamInputs.RECV_ALTERNATIVE_SERVICE):
         (None, StreamState.CLOSED),
 
-    # While closed, all other received frames should cause RST_STREAM
-    # frames to be emitted. END_STREAM is always carried *by* a frame,
-    # so it should do nothing.
+    # An endpoint that receives any frame other than PRIORITY after receiving
+    # a RST_STREAM MUST treat that as a stream error of type STREAM_CLOSED.
+    # An endpoint that receives any frames after receiving a frame with the
+    # END_STREAM flag set MUST treat that as a connection error of type
+    # STREAM_CLOSED.
```

Comment:
“RFC” should be capitalised, but otherwise that would be fine.

Example here: https://github.com/chhsiao90/hyper-h2/blob/9146feeae5443bcd5b4526e59bc8585c62c7adf4/h2/stream.py#L803